pages/pipelines/incoming_webhooks.md.erb

```diff
@@ -13,7 +13,26 @@ Secrets are pipeline-specific. A secret can’t prove that it hasn’t been acqu
 
 You can also provide the permitted IPs for GitHub Enterprise Server, BitBucket Server, GitLab Community, and GitLabEnterprise in your [organization's settings](https://buildkite.com/organizations/~/settings). To enable IP filtering for them, in the [Repository providers](https://buildkite.com/organizations/krnswry/repository-providers) tab, enter the allowed IP addresses. 
 
-For non-server BitBucket, IP filtering for webhooks is currently unavailable.
+For non-server BitBucket, IP filtering for webhooks is currently unavailable.  
```

Comment:
In the previous paragraph we let a `krnswry` slip through in the URL.

Reply:
Hosted BitBucket maybe? I know that isn't included in this PR, but that read awkwardly.  Alternatively we could go for IP filertering is only availabl for:
* X
* Y
* Z

Reply:
/Also (sorry) these should be H2s right? Maybe with a `{:toc}`

Reply:
Implementing all the suggested changes & remaking the Qs into H2s - but I'm a bit afraid that it's going to wreck the formatting soooo hard.